src/node/mod.rs

```diff
@@ -248,11 +263,15 @@ impl sc_executor::NativeExecutionDispatch for ExecutorDispatch {
     }
 }
 
+// enum FarmerCommand {
+//     Stop(futures::channel::oneshot::Sender<()>),
+// }
+
```

Comment:
why are these commented out for now?

Reply:
Oopsie. Forgot to clean it up. Will do now